simulation_parameters/microbe_stage/biomes.json

```diff
@@ -97,6 +97,22 @@
                             "amount": 60000
                         }
                     }
+                },
+                "sporeCell": {
+                    "name": "SPORE_CELL",
```

Comment:
I think maybe we should avoid the word "Spore" being in the game at all, so this could be something like "GOOGLY_EYED_CELL" or some kind of nickname like that.